gatsby/src/pages/__tests__/404.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { renderWithRouter } from "test-utils";
+import NotFoundPage from "../404";
+
+describe("404", () => {
+	it("should match snapshot", () => {
+		const { container } = renderWithRouter(<NotFoundPage />);
+
+		expect(container.firstChild).toMatchSnapshot();
+	});
+
+	it("should render h1 with correct text", () => {
```

Comment:
We could probably do with a test that asserts that the noindex robots tag is rendered

Reply:
done

Reply:
Unresolving this as I still think we need a test for this component - we have a test for the SEO compnent itself but right now we could remove `noIndex={true}` from the 404 component and no test would break, so we need something asserting that the meta tag is rendered

Reply:
I've added a test but it fails. I can't get the SEO to appear in the snapshot, so don't know what to try next. It also doesn't appear in the index snapshot. Help please.